src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
@paulomorgado When `invocationExpression.AccessorList` is null:

`invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1` returns `true`

`invocationExpression.AccessorList?.Accessors.Count == 1`returns `false`. That is wrong.